README.md

```diff
@@ -66,17 +66,18 @@ This will start a jupyter session, with several example notebooks detailing vari
 
 DeepCell uses `nvidia-docker` and `tensorflow` to enable GPU processing.
 
-### Build a local docker container
+If using GCP, there are pre-built images which come with CUDA, docker, and nvidia-docker pre-installed: 
+https://console.cloud.google.com/marketplace/details/nvidia-ngc-public/nvidia_gpu_cloud_image
 
-```bash
-git clone https://github.com/vanvalenlab/deepcell-tf.git
-cd deepcell-tf
-docker build -t $USER/deepcell-tf .
-```
+Otheriwise, these will need to be downloaded and installed:
+https://github.com/NVIDIA/nvidia-docker
+https://docs.docker.com/install/linux/docker-ce/debian/
```

Comment:
Can you please format the link into the text? I don't want to see the whole URL.